snippets/languages.mdx

```diff
@@ -0,0 +1,85 @@
+## Language configuration
+
+### Single language
+
+If you know the language of the conversation in advance, specify it in the `language_config.languages` parameter to ensure the best transcription results.
+
+```json
+{
+  ...rest of the config
+  "language_config": {
+    "languages": ["en"]
+  }
+}
+```
+
+If the spoken language is unknown, you can:
+
+- Omit the `language_config.languages` parameter; the model will automatically detect the language from the first few seconds of audio **across all supported languages**.
+- Specify multiple languages in the `language_config.languages` parameter; the model will detect the language from the first few seconds of audio **within the provided options**.
```

Comment:
Since the detection algo is different between pre-recorded and streaming, maybe omit the `from the first few seconds` ?  

```suggestion
- Omit the `language_config.languages` parameter; the model will automatically detect the language from the audio **across all supported languages**.
- Specify multiple languages in the `language_config.languages` parameter; the model will detect the language from the audio **within the provided options**.
```